automations/python/generate_report.py

```diff
@@ -0,0 +1,50 @@
+"""Python script for generating report for workflow"""
+
+import json
+import os
+import sys
+from collections import defaultdict
+
+
+server_url = os.environ.get("SERVER_URL")
+repository = os.environ.get("REPOSITORY")
+run_id = os.environ.get("RUN_ID")
+
+jobs = ["emit-docs", "publish-images", "deploy-frontend", "deploy-api"]
+
+results = {}
+counts = defaultdict(lambda: 0)
+
+for job_name in jobs:
+    result = os.environ.get(f"{job_name.replace('-', '_')}_result".upper())
+    results[job_name] = result
+    counts[result] += 1
+
+payload = {
+    "text": ", ".join(f"{count} {result}" for result, count in counts.items()),
+    "blocks": [
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": f"<{server_url}/{repository}/actions/runs/{run_id} \
+                |Click here to review \
+                the completed CI + CD workflow>.",
```

Comment:
Prefer to use parenthesis for longer strings over `\`. Somethign like this would be more readable.

```suggestion
                "text": (
                    f"<{server_url}/{repository}/actions/runs/{run_id}|"
                    "Click here to review the completed CI + CD workflow>."
                ),
```